xarray/core/dataarray.py

```diff
@@ -1094,6 +1094,7 @@ def chunk(
         self,
         chunks: Union[
             int,
+            str,
```

Comment:
How about a literal here? https://mypy.readthedocs.io/en/stable/literal_types.html

Reply:
Gonna need some compatibility code for 3.7 though, but copying something like this will probably do the trick:
https://github.com/pydata/xarray/blob/b449945f4f90628d0abff9b58e7312ad421eb97d/xarray/core/options.py#L6-L12